qa/L0_model_config/test.sh

```diff
@@ -239,6 +240,11 @@ mkdir -p autofill_noplatform_success/tensorrt/reshape_config_provided/1
 cp /data/inferenceserver/${REPO_VERSION}/qa_reshape_model_repository/plan_zero_4_float32/1/model.plan \
     autofill_noplatform_success/tensorrt/reshape_config_provided/1
 
+# Copy identity model into onnx test directories
+mkdir -p autofill_noplatform_success/onnx/cpu_instance/1
+cp -r /data/inferenceserver/${REPO_VERSION}/qa_identity_model_repository/onnx_zero_1_float16/1/model.onnx \
+    autofill_noplatform_success/onnx/cpu_instance/1
```

Comment:
Do the same for the TF savedmodel?

Reply:
I might push back on this as all other savedmodels are being stored in their respective test dir. I think it would make the test directory less intuitive to have one test pull from the model repository and the rest don't. I can file a ticket for cleanup here if this is what we want moving forward?

Reply:
I see your point. Personally I would want to use model from model repository if possible, then if we need to understand the model, we can track back to the generation script. I guess you can still examine the model file directly to understand its structure, but you will need to use some tools to read in the model.